src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  --text-blue: #00acdc;
```

Comment:
Create a variable with root selector

Reply:
If I understand correctly, you mean it's not a normal name for a variable.